.gitignore

```diff
@@ -4,3 +4,4 @@
 /.phpunit.cache
 /build/
 /coverage.clover
+.history
```

Comment:
This should go into your global gitignore as it's not related to the project but your local environment.

https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Removed